### ISSUE TITLE
Full Text Search as a plugin

### ISSUE THREAD

**Issue by wohali** [apache/couchdb#1516]
Once #1515 is done, this should be easy.

@janl:
>* deeeper clouseau/dreyfus integration
>* elasticsearch?

@davisp:
> At least a clear example of how to bundle and include Clouseau/Dreyfus.
> Just had someone ask how to use Dreyfus/Clouseau with CouchDB and I had to google for instructions and came across this:
>
>https://developer.ibm.com/dwblog/2015/text-search-apache-couchdb/
>
>Definitely Exhibit A on needing better integration. That's a lot of difficult steps for getting full text set up.

**Comment by wohali** [comment]
This is a duplicate of #1269 as it turns out.
